### PR TITLE
Feature/phpcs

### PR DESCRIPTION
**Pull request by defunctl**
## What does this do/fix?

More modern and strict PHPCS rules to play with. Leaving this in draft for a while so we can collect some feedback, and maybe others can contribute some comment spacing styles so they can be standard across all projects as well.

## Tests

Does this have tests?

- [ ] Yes
- [x] No, this doesn't need tests because these are just PHPCS rules
- [ ] No, I need help figuring out how to write the tests.

### CODE REVIEW

File: phpcs.xml [hunk 1]
```diff
@@ -41,41 +45,106 @@
...
         <exclude name="WordPressVIPMinimum.Constants.ConstantString.NotCheckingConstantName"/>
         <exclude name="WordPressVIPMinimum.Variables.VariableAnalysis"/>
     </rule>
 
+    <!-- Generic -->
     <rule ref="Generic.Functions.OpeningFunctionBraceKernighanRitchie"/>
     <rule ref="Generic.Classes.OpeningBraceSameLine"/>
     <rule ref="Generic.Arrays.DisallowLongArraySyntax.Found"/>
     <rule ref="Generic.WhiteSpace.DisallowSpaceIndent"/>
+    <rule ref="Generic.WhiteSpace.DisallowSpaceIndent"/>
```
Comment (lindseyolson) [inline phpcs.xml:81]: I think this is a repeat of line 80

Comment (defunctl) [reply phpcs.xml]: Thank you @lindseyolson!

**Comment by tarecord** [comment]
Once we've worked this out to something everyone agrees on, do you plan to update the whole project to follow these new rules at once? Or is that something we work on as we go?

**Comment by tarecord** [comment]
Confirmed that formatting also works with VS Code using [PHP Sniffer & Beautifier](https://marketplace.visualstudio.com/items?itemName=ValeryanM.vscode-phpsab).

Screen Recording 2021-05-20 at 12 46 40 PM 2021-05-20 12_50_31

**Comment by defunctl** [comment]
Nice, I'll try to open a "transition" PR that doesn't have the complete strict rules but just formatting updates so we can push that and fix all the files automatically and bring this in some time later.

**Comment by defunctl** [comment]
Closing in favor of https://github.com/moderntribe/square-one/pull/739

**Pull request closed without merging**